azavea/pfb-network-connectivity
# Add data migration to remove states from labels

## Overview

Sub-PR for #468, adding a data migration to rewrite existing labels without the state abbreviation.

## Testing Instructions

- Run migrations
- If it fails because there are neighborhoods that aren't unique on `(name, state, organization)`, which might be the case in your development database if you made a bunch of similar neighborhoods for debugging, remove the extras and try again.
- Confirm that your neighborhoods have their proper labels which no longer include redundant state abbreviations.

Connects #449.
